eng/dockerfile-templates/runtime-deps/Dockerfile.chiseled-ubuntu

```
@@ -27,27 +32,30 @@ RUN {{InsertTemplate("Dockerfile.linux.non-root-user-staged", [
         "create-home": "true"
     ])}}
 
-COPY --from=chisel /opt/chisel/chisel /usr/bin/
-RUN chisel cut --release "ubuntu-{{osVersionNumber}}" --root /rootfs \
-        base-files_base \
-        base-files_release-info \
-        ca-certificates_data \
-        libc6_libs \
-        libgcc-s1_libs \
-        libssl3_libs \
-        libstdc++6_libs \
-        zlib1g_libs
+COPY --from=chisel /opt/chisel/chisel /opt/rocks-toolbox/chisel-wrapper /usr/bin/
+
+RUN mkdir -p /rootfs/var/lib/dpkg/ \
+    && chisel-wrapper --generate-dpkg-status /rootfs/var/lib/dpkg/status -- \
+        --release "ubuntu-{{osVersionNumber}}" --root /rootfs \
+            base-files_base \
+            base-files_release-info \
+            ca-certificates_data \
+            libc6_libs \
+            libgcc-s1_libs \
+            libssl3_libs \
+            libstdc++6_libs \
+            zlib1g_libs
```

Looking at our other Dockerfiles, I see that we indent 4 spaces for package names.
```suggestion
        base-files_base \
        base-files_release-info \
        ca-certificates_data \
        libc6_libs \
        libgcc-s1_libs \
        libssl3_libs \
        libstdc++6_libs \
        zlib1g_libs
```

---

My intention was to treat the text after the `--` as another command (it's what is passed to the chisel cut command by the wrapper) and thus have them indented one more stop. I thought that having the `chisel cut` arguments on the same tab stop as the slices made it more difficult to read:

```Dockerfile
RUN mkdir -p /rootfs/var/lib/dpkg/ \
    && chisel-wrapper --generate-dpkg-status /rootfs/var/lib/dpkg/status -- \
        --release "ubuntu-{{osVersionNumber}}" \
        --root /rootfs \
        base-files_base \
        base-files_release-info \
        ca-certificates_data \
        libc6_libs \
        libgcc-s1_libs \
        libssl3_libs \
        libstdc++6_libs \
        zlib1g_libs
```